Src/Support/Google.Apis.Core/Http/HttpClientFactory.cs

```diff
@@ -56,14 +56,17 @@ protected virtual HttpMessageHandler CreateHandler(CreateHttpClientArgs args)
             // - Compression requested but not supported by HttpClientHandler (easy; just GzipDeflateHandler on top of an interceptor on top of HttpClientHandler)
             // - Compression requested and HttpClientHandler (complex: create two different handlers and decide which to use on a per-request basis)
 
-            var clientHandler = CreateAndConfigureClientHandler();
+            var clientHandler = CreateClientHandler();
+            var httpClientHandler = GetHttpClientHandler(clientHandler);
```

Comment:
I don't like this because it may cause maintenance problems. Now we need to remember to use the `httpClientHandler` for configuration but `clientHandler` for the rest, else, we've just stripped out all delegating handlers that were set originaly on `clientHandler`.

Reply:
Yes, but it is covered in tests. It is not a big issue then.